### PR TITLE
Release/2.2.0

### PR DESCRIPTION
**Pull request by FreemanPancake**
## Description
Bump version for django 4.2 support

## Related resources

* #...
* #...

## Checklist

* [x] I have added or modified the tests when changing logic
* [x] I have followed [the conventional commits guidelines](https://www.conventionalcommits.org/) to add meaningful information into the changelog
* [x] I have read the [contribution guidelines ](https://github.com/django-cms/django-cms/blob/develop/CONTRIBUTING.rst)

### CODE REVIEW

**Review by fsbraun — APPROVED** [review]

File: tests/test_monkeypatch.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def test_get_edit_link(self, mock_is_obj_review_locked):
         )
         # We test that moderation check is called when getting an edit link
         self.assertTrue(mock_is_obj_review_locked.called)
-        if versioning_version < "2":
+        if versioning_version != "2.0.0":
             # Edit link is inactive as `mock_is_obj_review_locked` is True
             self.assertIn("inactive", edit_link)
         else:
```
Comment (fsbraun) [inline tests/test_monkeypatch.py:48]:
```suggestion
        self.assertIn("inactive", edit_link)
```

**plus** deleting the next three lines

File: tests/test_monkeypatch.py [hunk 2]
```diff
@@ -92,7 +92,7 @@ def test_get_archive_link(self, _mock):
             archive_link = ""
         # We test that moderation check is called when getting an edit link
         self.assertEqual(1, _mock.call_count)
-        if versioning_version < "2":
+        if versioning_version != "2.0.0":
             # Edit link is inactive as `mock_is_obj_review_locked` is True
             self.assertIn("inactive", archive_link)
         else:
```
Comment (fsbraun) [inline tests/test_monkeypatch.py:98]:
```suggestion
        self.assertIn("inactive", archive_link)
```
**plus** deleting the next three lines

**Comment by joshyu** [comment]
@fsbraun  / @FreemanPancake ,
It's so weird that I run pre-commit locally on all files and every check passed.
image

**Comment by FreemanPancake** [comment]
Hi @fsbraun ,
I have update the error fix for pre-commit please review.

**Pull request merged**